test/CMakeLists.txt

```diff
@@ -323,6 +323,93 @@ if(AMR_WIND_ENABLE_W2A)
+      WORKING_DIRECTORY
+      "${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND ${CMAKE_COMMAND} -E create_symlink
+      ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${AERO_FILE}
+      ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_BASELINE_DIR}/${AERO_FILE}
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND ${CMAKE_COMMAND} -E create_symlink
+      ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${ELASTO_FILE}
+      ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_BASELINE_DIR}/${ELASTO_FILE}
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND ${CMAKE_COMMAND} -E create_symlink
+      ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}
+      ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_BASELINE_DIR}/${SERVO_FILE}
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/5   PCMode/0   PCMode/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/5   VSContrl/1   VSContrl/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/9999.9   VS_RtGnSp/1161.963   VS_RtGnSp/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/9999.9   VS_RtTq/43093.55   VS_RtTq/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/9999.9   VS_Rgn2K/0.0255764   VS_Rgn2K/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/9999.9   VS_SlPc/10.0   VS_SlPc/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${SERVO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/1   WakeMod/0   WakeMod/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${AERO_FILE}"
+      WORKING_DIRECTORY
+      "${ACT_UNIFORM_ALM_TEST_BINARY_DIR}"
+      COMMAND_ERROR_IS_FATAL ANY)
+    execute_process(
+      COMMAND bash -c "sed -i -e 's/True          YawDOF/False         YawDOF/g' ${RTEST_ACT_UNIFORM_ALM_TEST_BINARY_DIR}/${5MW_LAND_DIR}/${ELASTO_FILE}"
```

Comment:
Unfortunately sed isn't portable.

Reply:
yeah... got another idea?

Reply:
I actually thought of a better way. We carry a patch and `git apply` it. Added benefit of this is that if the patch doesn't apply due to file changes in openfast, we know about it immediately.

Reply:
That's a great idea.